app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -128,6 +128,10 @@ def track_document_request_event(document_request:, document_response:, timer:)
       analytics.idv_socure_document_request_submitted(**analytics_hash)
     end
 
+    def choose_id_type_path
+      'choose_id_type'
```

Comment:
If we do this, we're protected from any future changes in the path. One potential issue is if it could also be used on the ipp side - that would be `idv_in_person_choose_id_type_path`

```suggestion
      idv_choose_id_type_path
```